circe-akka-serializer/src/main/scala-2.12/org/virtuslab/ash/CollectionConverter.scala

```diff
@@ -0,0 +1,8 @@
+package org.virtuslab.ash
+import java.util
+import scala.collection.convert.AsScalaConverters
+import scala.collection.mutable
+
+object CollectionConverter extends Converter with AsScalaConverters {
```

Comment:
Nit: isn't there some scala-collection-compat library that does these things? or maybe I confused its purpose

Reply:
Oh okay, I see the purpose of this construct, let's leave as is then

Reply:
Actually, `scala-collection-compat` solved this problem. The question is, do we want to include it as our dependency?

Reply:
Hmm does it have any deps on its own and/or is it likely to introduce compatibility problems (e.g. if multiple versions of `scala-collection-compat` aren't compatible with each other)?

Reply:
From project README:
> All future versions will remain backwards binary compatible with 2.0.0. (The 1.0.0 release was withdrawn and should not be used.)

One problem can be found [here](https://github.com/VirtusLab/akka-serialization-helper/blob/db10808b5e52e1a2adff229035ece6f7b349ac95/build.sbt#L124). Manageable, I dare say.